proto/bootz.proto

```diff
@@ -136,13 +154,22 @@ message BootstrapStreamResponse {
       // The public key used to encrypt the data will be based on the 
       // public key provided by OVGS. This maybe EK or PPK.
       bytes nonce_encrypted = 2; 
+      // TPM 1.2 EK CA public cert
+      bytes ca_pub = 3;
     } 
   } 
   
+  message EKIdentityResponse {
+    // TPM 1.2 challenge
+    bytes asymetric = 1;
```

Comment:
Typo here: Should be assymetric and symmetric

Reply:
Fixed it